ember-cli-build.js

```diff
@@ -44,7 +44,7 @@ function resolveRelativeModulePath(name, child) {
 
 module.exports = function (defaults) {
   let checker = new VersionChecker(defaults);
-  let emberChecker = checker.forEmber();
+  let emberChecker = checker.for('ember');
```

Comment:
I think this needs to be `.for('ember-source')` right?

Reply:
Ah, perhaps. I had seen `let dep = checker.for('ember', 'bower');` in the docs.